v3/plugins/vscodevim/vim/v1.12.0/meta.yaml

```diff
@@ -0,0 +1,15 @@
+publisher: vscodevim
+apiVersion: v2
+name: vim
+version: v1.12.0
```

Comment:
Minor nitpick/optional request: the other plugins generally avoid using `v` in version numbers. I'd prefer to stick to a `major.minor.bugfix` format where possible (in case we need to process version numbers at some later point).

Reply:
fixed